### DISCUSSION TITLE
Game Title

### DISCUSSION THREAD

**Discussion by Jumalita** [2WolfGames/Erlang-Legacy#189] [Show and tell]
I have been working on the game title.

Which one do you like more @wilberquito  ?
![imagen](https://user-images.githubusercontent.com/61147274/173920786-6d74b046-387c-4756-8352-b31f5dd118a0.png)

**Comment by wilberquito** [comment]
@Jumalita I like 1 and 4

**Comment by Jumalita** [comment]
Janira: 3 o 4

**Comment by Jumalita** [comment]
![imagen](https://user-images.githubusercontent.com/61147274/174165245-e42b45d1-929a-4821-9933-e8974df23538.png)

**Reply by Jumalita** [reply]
Another one
@wilberquito

**Comment by Jumalita** [comment]
We have choose the last one.
